### PR TITLE
Lock grcov version

### PR DESCRIPTION
**Pull request by markcty**

### CODE REVIEW

File: .github/scripts/install_deps.sh [hunk 1]
```diff
@@ -1,4 +1,11 @@
 apt-get install -y cmake g++
+
+# install protoc
 wget https://github.com/protocolbuffers/protobuf/releases/download/v21.10/protoc-21.10-linux-x86_64.zip
 unzip protoc-21.10-linux-x86_64.zip -d .local
-mv "$(pwd)/.local/bin/protoc" /bin/
```
Comment (Phoenix500526) [inline .github/scripts/install_deps.sh:4]: Please add a new line at the end of this file. Consistency with POSIX standards

File: .github/scripts/install_deps.sh [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
\ No newline at end of file
+mv "$(pwd)/.local/bin/protoc" /bin/
+
+# install grcov
+wget https://github.com/mozilla/grcov/releases/download/v0.8.13/grcov-x86_64-unknown-linux-gnu.tar.bz2
+tar -xjvf grcov-x86_64-unknown-linux-gnu.tar.bz2
+mv grcov /bin/
```
Comment (Phoenix500526) [inline .github/scripts/install_deps.sh:11]: Add the new line whose line number is 12.

Comment (markcty) [reply .github/scripts/install_deps.sh:11]: There is a new line. Github hides it.

Comment (Phoenix500526) [reply .github/scripts/install_deps.sh:11]: OK, I see

**Review by Phoenix500526 — APPROVED** [review]

**Review by themanforfree — APPROVED** [review]

**Pull request merged**